source/geod24/concurrency.d

```diff
@@ -107,8 +77,8 @@ private
 
     @safe unittest
     {
-        static struct Container { Tid t; }
-        static assert(!hasLocalAliasing!(Tid, Container, int));
+        static struct Container { Channel!int t; }
+        static assert(!hasLocalAliasing!(Channel!int, Container, int));
```

Comment:
```suggestion
        static assert(!hasLocalAliasing!(Channel!(Channel!int), Channel!int, Container, int));        
        static assert( hasLocalAliasing!(Channel!(Channel!(int[]))));        
```

For testing the new code path ?

Reply:
and it breaks immediately:)

Reply:
We're passing locally aliased types around ?

Reply:
`hasLocalAliasing!(Channel!(Channel!int))`

It didnt like the nested channels:)

```source/geod24/concurrency.d(66,24): Error: declaration R is already defined```

Reply:
Hum, try replacing `R` with something else... E.g. `ChannelType` ?

Reply:
https://issues.dlang.org/show_bug.cgi?id=20018